ethpm_types/contract_type.py

```diff
@@ -324,9 +324,34 @@ def get_deployment_bytecode(self) -> Optional[HexBytes]:
 
         return None
 
+    @cached_property
+    def identifier_definitions(self) -> Dict[str, str]:
+        """
+        Returns a mapping of the full suite of signatures to selectors/topics/IDs for this
+        contract
+        """
+
+        def get_id(aitem: ABI) -> str:
+            if isinstance(aitem, MethodABI) or isinstance(aitem, ErrorABI):
+                return HexBytes(self._selector_hash_fn(aitem.selector)[:4]).hex()
```

Comment:
note: I think there is also `self._selector_id_size` if you wish to utilize that.

Reply:
I don't think this is available here?  Looks like it's part of ABIList and the list I'm pulling from here is various.  So unless I pull each one individually I don't think I have hat.  Unless I'm missing something?

Reply:
Weird, I thought if `_selector_hash_fn` was available then `_selecor_id_size` would be too.

Reply:
We may want the selector size to be configurable some day.

Reply:
Might be cleaner.  Though I'd be shocked if that ever changed.